CMakeLists.txt

```diff
@@ -96,6 +96,20 @@ if(IS_FEDORA_LIKE)
   endif(DEFINED ENV{MPI_SUFFIX})
 endif(IS_FEDORA_LIKE)
 
+# Check if it is conda environment
+set(IS_CONDA false)
+if(DEFINED ENV{CONDA_PREFIX})
```

Comment:
This only works for genuine Conda environments. But how about non-standard, Conda-based environments, such as Colab (where `CONDA_PREFIX` is not defined)? Native environment will be assumed, which does not seem quite accurate IMHO (because `bin` is still scoped by a Conda environment path).